docs/contributing.rst

```diff
@@ -94,6 +94,12 @@ pip::
   cd sgkit
   pip install -r requirements.txt -r requirements-dev.txt -r requirements-doc.txt
 
+You also need to install GSL (GNU Scientific Library), which is required for ``msprime``.
+Since its a system package and installation may vary depending on the operating
+system, the easiest way to install it is using conda::
+
+    conda install -c conda-forge gsl
```

Comment:
It feels a bit weird that we first recommend using pip above and then after it likely fails on `msprime` install, we bring in `conda`. I wonder if we should instead link to [msprime doc](https://msprime.readthedocs.io/en/stable/installation.html#installing-system-requirements) which has extensive information about GSL install options?

Reply:
True, that is weird indeed. I'll add a link to msprime doc rather.